packages/server/src/watch.ts

```diff
@@ -0,0 +1,34 @@
+import {checkMatch} from "@prodo-ai/snoopy-search";
+// tslint:disable-next-line:no-submodule-imports
+import {ignored} from "@prodo-ai/snoopy-search/src/utils";
```

Comment:
Will this work when it's built (and the code will live in `lib/` instead)? 

Why not just expose `ignored` as part of the `@prodo-ai/snoopy-search`s API?